tests/rbd_mirror/test_mirror_move_secondary_trash.py

```diff
@@ -0,0 +1,86 @@
+from tests.rbd.rbd_utils import Rbd
+from tests.rbd_mirror.rbd_mirror_utils import rbd_mirror_config
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def test_image_move_secondary_trash(rbd_mirror, pool_type, **kw):
+    """
+    Test to check secondary image move to trash is failed as image will be locked and busy.
+    Args:
+        rbd_mirror: rbd mirror object
+        pool_type: ec pool or rep pool
+        **kw:
+    Returns:
+        0 if test pass, else 1
+    """
+    try:
+        mirror1 = rbd_mirror.get("mirror1")
+        config = kw.get("config")
+        pool = config[pool_type]["pool"]
+        image = config[pool_type]["image"]
+        imagespec = pool + "/" + image
+
+        rbd1, rbd2 = [
+            Rbd(**kw, req_cname=cluster_name)
+            for cluster_name in kw.get("ceph_cluster_dict").keys()
+        ]
+
+        mirror1.benchwrite(imagespec=imagespec, io=config.get("io-total", "1G"))
```

Comment:
Unless we have strong reason, can we remove configs from suite and use defaults ?

I mean we remove 
imagesize: 2G
io-total: 200M
from suites ad change io=config.get("io-total", "1G") to io=config.get("io-total", "200M") is 200M is really desired io amount otherwise we can reduce 3*2 (suites) lines of code no?